### ISSUE TITLE
Support for StackSet resources

### ISSUE THREAD

**Issue by benkehoe** [aws/aws-sam-cli#2249]
CloudFormation now has an [AWS::CloudFormation::StackSet](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cloudformation-stackset.html) resource. The SAM CLI should add local path support for the `TemplateURL` parameter, allowing the template for a StackSet to be defined in a separate file, and doing the normal upload-to-S3-and-insert-URL packaging process.

**Comment by prowe-athene** [comment]
I have added support for this here: https://github.com/aws/aws-sam-cli/pull/3249

**Comment by maintux** [comment]
Hi @prowe-athene, your PR seems not working. Eg.
```
$> sam package -t test.yaml --s3-bucket xxxxxxx
AWSTemplateFormatVersion: 2010-09-09
Transform: AWS::Serverless-2016-10-31
Resources:
  rBackupVaultStackSet:
    Type: AWS::CloudFormation::StackSet
    Properties:
      StackSetName: testss
      PermissionModel: SELF_MANAGED
      StackInstancesGroup:
      - DeploymentTargets:
          Accounts:
          - Ref: AWS::AccountId
        Regions:
        - Ref: AWS::Region
      TemplateURL: ./ss.yaml
    Metadata:
      SamResourceId: rBackupVaultStackSet

**Comment by hawflau** [comment]
PR merged. waiting for release

**Comment by qingchm** [comment]
Hey there @benkehoe the support for this has been released in our SAMCLI v1.44.0, please update to get the feature! Thanks for contacting and thank you @maintux and @prowe-athene for active contributions on this issue! Really appreciate it, closing the issue now and feel free to contact us for any further questions!
